### ISSUE TITLE
plv8 for Posgres 14

### ISSUE THREAD

**Issue by neemah** [PostgresApp/PostgresApp#644]
```
create extension plv8;
ERROR:  could not open extension control file "/Applications/Postgres.app/Contents/Versions/14/share/postgresql/extension/plv8.control": No such file or directory
```

plv8 not included in postgres 14, or I did something wrong?

Sorry if this not an appropriate place to ask that questions, did not find chat or related issues.

Thank you

**Comment by tbussmann** [comment]
You are right, plv8 is unfortunately no longer included in our builds of PostgreSQL 14 onwards. You can find more information at https://github.com/PostgresApp/PostgresApp/issues/602#issuecomment-932550026 I haven't looked into it for a while, but it may be possible to build the extension on your own if you need it.

**Comment by ppKrauss** [comment]
No news?  Feb 2022

**Comment by jakob** [comment]
@ppKrauss Nothing has changed.

We have no plans to include plv8 with PostgreSQL 14 and later.

We are of course still including it with old versions of PostgreSQL (up to PostgreSQL 13) because we don't want to break existing installations.

**Comment by braco** [comment]
Ah that's too bad, anecdote of one here but this was a critical feature.

@jakob from your linked thread:

> we look at what Linux distros are doing, and what cloud providers are doing -- as far as I know none of them include plv8 in their distributions anymore, probably because it is too difficult to package and because it is not possible to guarantee long time support.

Which cloud providers dropped support?

RDS: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.PostgreSQL.CommonDBATasks.Extensions.html#PostgreSQL.Concepts.General.UpgradingPLv8

**Comment by jakob** [comment]
> Which cloud providers dropped support? [RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.PostgreSQL.CommonDBATasks.Extensions.html#PostgreSQL.Concepts.General.UpgradingPLv8), [supabase](https://supabase.com/docs/guides/database/extensions/plv8)

I remember that Heroku [dropped plv8](https://devcenter.heroku.com/changelog-items/1677), but it seems that I was wrong about extrapolating from this -- cloud providers have not all dropped plv8. Another example would be Neon, they [also supports plv8](https://neon.tech/docs/reference/pg-extensions/).

It's a bit easier for cloud providers, since they only have to get it working on their own machines. When we add something to Postgres.app, we try to keep supporting it for ~5 years across as many versions of macOS as possible.

Anyway, the biggest reason why we dropped plv8 is that it took extremely much effort to build v8, and I'm not sure if anything has improved on that front.
